source/html/index.html

```diff
@@ -126,16 +126,16 @@ <h2 id="editTask">Editing TODO specific Name</h2>
           /><br />
           <label for="statusField">Status: </label>
           <select name="status" id="statusField">
-            <option value="planned">Planned</option>
-            <option value="inProgress">In-Progress</option>
-            <option value="complete">Complete</option>
+            <option value="Planned">Planned</option>
+            <option value="In-Progress">In-Progress</option>
+            <option value="Complete">Complete</option>
           </select>
           <div class="textArea">
             <label for="noteFieldEdit">Notes: </label>
             <textarea id="noteFieldEdit" name="note" required></textarea><br />
           </div>
           <button type="button" id="cancelEditButton">Cancel</button>
-          <button class="submitButton" id="subEditButton">Save</button>
+          <button class="submitEditButton" id="subEditButton">Save</button>
```

Comment:
I don't think we need the `id` here since they are not used in the javascript (only the class names were called) and the ids get redefined to the corresponding task ids with the unique hash code.

Reply:
Good catch, I double checked and they were not used. I also removed the id of the deleteEditButton as well since it was not used. It is probably not a good idea to have the ID as the same name as the class name also.